V1.0/sb hotfix/add ticketfrom i detc

sborjas opened the pull request.

sborjas committed:
popup adds

sborjas committed:
utf8 y mas detalles, cambios agregados

sborjas committed:
Detalles arreglados

Se corrigio el problema que habia al recibir error al enviar ticket, el loading se quedaba pegado.
Ya se pueden enviar tickets con caracteres especiales. (Aportacion Ivan)
Ya arregle la implementacion del codigo de David diaz (agregar ticket desde ID), la absolute layout tapada toda la listview, por lo tanto no permitia mostrar los ticekts.

sborjas committed:
gitignore..

damasofc reviewed (changes requested):
-Cuando agrego un ticket que no existe, igual me dice que el ticket ha sido agregado.
-y de igual forma sucede cuando agrego un ticket que no esta a mi nombre.
- constantemente esta apareciendo el loading de tickets, solo debe aparecer cuando carga la primera vez, y cuando uno da swipe para recargar.

sborjas committed:
se ha corregido lo detallado por damaso...

-Cuando agrego un ticket que no existe, igual me dice que el ticket ha sido agregado.
-y de igual forma sucede cuando agrego un ticket que no esta a mi nombre.
